### ISSUE TITLE
`_xxinterpqueues.get` return strange value, when the queue is empty

### ISSUE THREAD

**Issue by sobolevn** [python/cpython#116239]
# Bug report

Code:

```python
» ./python.exe                                      
Python 3.13.0a4+ (heads/main:5dc8c84d397, Mar  2 2024, 10:55:34) [Clang 15.0.0 (clang-1500.0.40.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import _xxinterpqueues
>>> a = _xxinterpqueues.create(2, 9)
>>> _xxinterpqueues.get(a, 'a')
('a', 0)
>>> 
```

It is happening on a fresh build of `main` after https://github.com/python/cpython/issues/116102 is fixed.

I can reproduce this 100% of times, `QueueEmpty` as @ericsnowcurrently said in https://github.com/python/cpython/issues/116102#issuecomment-1974100841 never happens to me.

I am on `macos`.

**Comment by Eclips4** [comment]
Seems this behaviour is intentional. Second parameter on `get` method means default value. So, it's ok to not to see the `QueueEmpty` exception. But, why is there a tuple as a return value of `get`?  The second item in this tuple looks superfluous (Should there really be a tuple here?)

**Comment by sobolevn** [comment]
> (Should there really be a tuple here?)

Yes, the tuple is of `(default, fmt)`. I still don't understand why `fmt` is `3` though

**Comment by ericsnowcurrently** [comment]
This should be cleared up by gh-116328.

**Comment by Eclips4** [comment]
This was fixed in #116328.
